test/config-map.json

```diff
@@ -206,9 +206,7 @@
     ],
     "image-types": [
       "edge-raw-image",
-      "edge-ami",
```

Comment:
Why was edge-ami removed from this config?

Reply:
removed `edge-ami` as the the `create-mountpoint.service` will fail on first boot resulting in `test/scripts/base-host-check.sh` to fail too, which now I think can modify to make it state agnostic for this service.